docs/sources/features/datasources/graphite.md

```diff
@@ -97,10 +106,10 @@ variable with all possible values that exist in the wildcard position.
 You can also create nested variables that use other variables in their definition. For example
 `apps.$app.servers.*` uses the variable `$app` in its query definition.
 
-### Variable usage
+### Variable Usage
 
 You can use a variable in a metric node path or as a parameter to a function.
-![](/img/docs/v2/templated_variable_parameter.png)
+![variable](/img/docs/v2/templated_variable_parameter.png)
 
 There are two syntaxes:
```

Comment:
Same comment here as for templating docs. What should be the preferred syntax to use starting from v5.0?

Reply:
Skipping this for now - need to dig a bit deeper into what the preferred syntax should be.